bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/model/discovery/projects/JavaResourceInfo.java

```diff
@@ -275,7 +275,12 @@ private IPath searchForBazelSupportedStructure(ResourceEntry fileEntry) {
             // we cannnot search outside the package for "java" or "javatests" but we try here
             if ("java".equals(segment) || "javatests".equals(segment)) {
                 // use this and abort
-                return path.append(segment);
+                path = path.append(segment);
+                // java/resources
+                if ((segments.length > (i + 1)) && "resources".equals(segments[i + 1])) {
+                    path = path.append(segments[i + 1]);
+                }
+                return path;
```

Comment:
This change seems unrelated to `filegroup` support. Please remove it. The `java/resources` structure is something that we did not intend to support automatically because it's uncommon and IntelliJ does not support it either. We should change the examples instead to not use this.